vendor/k8s.io/client-go/util/cert/BUILD

```diff
@@ -13,6 +13,9 @@ go_library(
         "io.go",
         "pem.go",
     ],
+    data = [
+        "testdata/dontUseThisKey.pem",
+    ],
```

Comment:
hummm, running update-deps added this entry, which causes bazel test to fail?

Reply:
is this still an issue? that's not great.

Reply:
this is a mistake in upstream client-go. it shouldn't have ever added testdata on the go_library, only on the go_test.